### PR TITLE
fix: typescript any keywords in errors

### LINKED ISSUE #24: (JS-0323) Detected usage of the `any` type
## Description
The `any` type can sometimes leak into your codebase. TypeScript compiler skips the type checking of the `any` typed variables, so it creates a potential safety hole, and source of bugs in your codebase. We recommend using `unknown` or `never` type variable.

## Occurrences
There are 44 occurrences of this issue in the repository.

See all occurrences on DeepSource &rarr; [deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0323/occurrences/](https://deepsource.io/gh/lenstube-xyz/lenstube/issue/JS-0323/occurrences/)

### PR DESCRIPTION
**Pull request by zt4ff**
## Description

Closes #24 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by zt4ff** [comment]
I did not fix some of the changes cause I wasn't sure if they will break stuff. 

So I will start contributing more to this repo to understand the code well and fix these typescript issue later.

IS that fine with you @sasicodes

File: src/types/index.ts [hunk 1]
```diff
@@ -3307,5 +3307,12 @@ export type WorldcoinIdentity = {
...
   }
 };
-      export default result;
-    
\ No newline at end of file
+
+export default result;
+    
+
+export interface CustomErrorWithData extends Error {
```
Comment (sasicodes) [inline src/types/index.ts:3314]: can define this type in local.ts file

Comment (zt4ff) [reply src/types/index.ts]: @sasicodes 
done ✅

**Comment by sasicodes** [comment]
@zt4ff there is an error in one of the file after rebasing master. can you fix that?

image

**Comment by zt4ff** [comment]
Hi @sasicodes 
I have been away for a while
I'll review this error and make appropriate changes

**Pull request closed without merging**